src/content/h1/arguments/readme.md

```diff
@@ -108,8 +108,11 @@ Texture Quality= High
 
 For further information, please visit the timedemo FAQ at: http://halo.bungie.net/site/halo/features/hpcperformancefaq.html
 ```
-
-Note that it says `Particles= Off` even if your particles setting is otherwise High, suggesting some of these options are hardcoded. The Bungie FAQ link is now a 404 page.
+##### Notes
+* The output says `Particles= Off` even if your particles setting is otherwise High, suggesting some of these options are hardcoded. The Bungie FAQ link is now a 404 page.
+* Each time `-timedemo` is run, the output is *amended* to the output file, `./timedemo.txt`. Previous output results and any custom text added by a third party will not be deleted or overwritten by the next `-timedemo` output.
+* The duration of `-timedemo` is affected by the framerate.
+* The maps tested during `-timedemo` are hardcoded to `ui`, `a30`, `b30`, `c10`, and `d20`. However, replacing the stock maps with custom maps works as well.
```

Comment:
This information IMO would better belong in the table above in the page. This section is more just so people can see a sample file and have a couple notes about the output itself rather than what timedemo does. My suggestion would be to remove the `#### Notes` heading and turning the first two bullet points into a small paragraph, and taking these last two points and merging that with the info about the argument in the table.